timepiece/crm/views.py

```diff
@@ -3084,3 +3085,36 @@ def project_download_attachment(request, project_id, attachment_id):
         return HttpResponseRedirect(project_attachment.get_download_url())
     except:
         return HttpResponse('Project attachment could not be found.')
+
+def get_minding(request):
+    user = request.user
+    projects = Project.objects.filter(
+        point_person=user, status=4)[:3]
+    projects_json = []
+    for project in projects:
+        projects_json.append({
```

Comment:
Actually, since you are using this same serialization below, please do move that into a `to_json` method on the `class Project` in `models.py`.  Then all you have to do here is `projects_json.append(project.to_json())`.